File: add_two_numbers.py
def add_three_numbers(a: float, b: float, c: float) -> float:
    return a + b + c


def main() -> None:
    a = float(input("Enter first number: ").strip())
    b = float(input("Enter second number: ").strip())
    c = float(input("Enter third number: ").strip())
    print(f"Sum: {add_three_numbers(a, b, c)}")


if __name__ == "__main__":
    main()

File: AddTwoNumbers.java
import java.util.Scanner;

public class AddTwoNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first number: ");
        double firstNumber = scanner.nextDouble();

        System.out.print("Enter second number: ");
        double secondNumber = scanner.nextDouble();

        double sum = firstNumber + secondNumber;
        System.out.println("Sum = " + sum);

        scanner.close();
    }
}
